### PR TITLE
Add a few spelling corrections

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -31405,6 +31415,7 @@ speciffic->specific
 speciffically->specifically
 specifi->specific, specify,
 specifially->specifically
+specificaiton->specification
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:31418]: Please also add the plural

Comment (int-y1) [reply codespell_lib/data/dictionary.txt:31418]: done

**Review by luzpaz — APPROVED** [review]

**Comment by luzpaz** [comment]
@vikivivi do you mind reviewing PR ? TIA!

**Comment by vikivivi** [comment]
If you don't mind adding these misspelling as well? You do not need to add those with 0 hit.

```text
cerain->certain
cerainty->certainty

uncerain->uncertain
uncerainties->uncertainties
uncerainty->uncertainty
```
* https://github.com/search?q=cerain&type=code  11418
* https://github.com/search?q=cerainly&type=code  3732
* https://github.com/search?q=cerainties&type=code  0
* https://github.com/search?q=cerainty&type=code  83
* https://github.com/search?q=uncerain&type=code  627
* https://github.com/search?q=uncerainly&type=code  0
* https://github.com/search?q=uncerainties&type=code  84
* https://github.com/search?q=uncerainty&type=code  2114

```text
cetain->certain
cetainly->certainly
cetainties->certainties
cetainty->certainty

uncetain->uncertain
uncetainly->uncertainly
uncetainties->uncertainties
uncetainty->uncertainty
```
* https://github.com/search?q=cetain&type=code  31402
* https://github.com/search?q=cetainly&type=code  2293
* https://github.com/search?q=cetainties&type=code  1
* https://github.com/search?q=cetainty&type=code  261
* https://github.com/search?q=uncetain&type=code  1046
* https://github.com/search?q=uncetainly&type=code  3
* https://github.com/search?q=uncetainties&type=code  169
* https://github.com/search?q=uncetainty&type=code  497

```text
compelxes->complexes
compelxities->complexities
```
* https://github.com/search?q=compelxes&type=code  44
* https://github.com/search?q=compelxities&type=code  48

**Comment by int-y1** [comment]
> If you don't mind adding these misspelling as well? You do not need to add those with 0 hit.

Done. Didn't add `certainties` and `uncertainly` due to rareness.

**Comment by vikivivi** [comment]
LGTM

**Comment by luzpaz** [comment]
Thank you!

**Pull request merged**